src/main/java/com/rtb/projectmanagementtool/init/InitTestData.java

```diff
@@ -33,18 +33,16 @@
 
   // The tasks
 
+  private ArrayList<Key> entityKeys; // arrays to store entity keys for batch delete
```

Comment:
probably doesn't need to be a class member since this is a temporary list.   Plus servlets don't hold state anyways across requests